engine/configstore/supabaseLegacy.ts

```diff
@@ -67,67 +89,88 @@ interface AudienceFlag {
   __resolveType: string;
 }
 
-interface CatchAllConfigs {
-  handler: {
-    flags: AudienceFlag[];
-    __resolveType: "$live/handlers/routesSelection.ts";
-  };
-  __resolveType: "resolve";
-}
-
 const sectionToAccount: Record<string, string> = {
   "deco-sites/std/sections/configVTEX.global.tsx":
     "deco-sites/std/accounts/vtex.ts",
   "deco-sites/std/sections/configOCC.global.tsx":
     "deco-sites/std/accounts/occ.ts",
   "deco-sites/std/sections/configShopify.global.tsx":
     "deco-sites/std/accounts/shopify.ts",
+  "deco-sites/std/sections/configVNDA.global.tsx":
+    "deco-sites/std/accounts/vnda.ts",
+  "deco-sites/std/sections/configYourViews.global.tsx":
+    "deco-sites/std/accounts/yourViews.ts",
```

Comment:
What happens when we have a global config that's not here?

Reply:
needs to be added or it will not work, functions will start to receive `undefined`

Reply:
I can create a stub value that just receive and returns, should be enough